File: Challenges/C++/ex1.cpp
#include <iostream>
#include <cstdlib>

using namespace std;

int main(int argc, char** argv){
    	if(argc == 2 && string(argv[1]) == "-h"){
		cout << "Hunt for arithmetic done to the argument vector elements." << endl;
    	}else if(argc!=3){
		cout << "Usage: ./prog input" << endl;
		cout << "Hint: ./prog -h" << endl;
	}else{
		atoi(argv[1])+atoi(argv[2])==1950?cout << "Score!" << endl:cout << "Fail!" << endl;
	}
	return 0;
}

File: Challenges/Go/ex1.go
package main

import "os"
import "fmt"
import "strconv"

func main(){
	if len(os.Args) == 2 && os.Args[1] == "-h"{
		fmt.Println("Just some simple arithmetic.")
	}else if len(os.Args) != 3{
		fmt.Println("Usage: ./prog input input")
		fmt.Println("Hint: ./prog -h")
	}else{
    		a, err := strconv.Atoi(os.Args[1])
    		if err != nil{
      			fmt.Println("Error!")
      			os.Exit(3)
    		}
    		b, err := strconv.Atoi(os.Args[2])
		if err != nil{
      			fmt.Println("Error!")
      			os.Exit(3)
    		}
    		c := a + b
		if c == 1789{
			fmt.Println("Score!")
		}else{
			fmt.Println("Fail!")
		}
	}
}
